database/factories/Order/ItemFactory.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories\Order;
+
+use App\Models\Product;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends \Illuminate\Database\Eloquent\Factories\Factory<\App\Models\=Item>
+ */
+class ItemFactory extends Factory
+{
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        return [
+            'product_id' => Product::factory(),
```

Comment:
All the fields should be required order_id, quantity (min 1), unit_price (can be 0.0 by default) we can improve the db defaults values in a new alter migration